addons/website_livechat/tests/common.py

```diff
@@ -39,7 +41,11 @@ def setUp(self):
             'country_id': self.env.ref('base.de').id,
             'website_id': self.env.ref('website.default_website').id,
             'partner_id': self.env.ref('base.user_demo').partner_id.id,
-        }] + [visitor_vals]*self.max_sessions_per_operator)
+            'access_token': self.env.ref('base.user_demo').partner_id.id,
+        }] + [
+            dict(visitor_vals, access_token=secrets.token_hex(20))
+            for _ in range(self.max_sessions_per_operator)
```

Comment:
This is just one random example showing that the `access_token` is being confused with a unique cryptographic secret, akin to the session_id. In the current state of this PR it is not true anymore, it's merely a "visitor fingerprint", neither secret nor unique.

This has implications wherever the concept of visitor is used.

It doesn't really matter in this particular test, but you could avoid using `secrets` anyhow ^^

Reply:
Thought it was a fast and easy to read way to generate a random 40 char length hexa string.
Note that any string would be fine as long as it is 40 char long.

I didn't planed to initially use `token_hex` but went for `'%040x' % random.randrange(16**40)`, thought `token_hex` would be easier to read, I'll revert to the random way